src/documents/bulk_edit.py

```diff
@@ -325,3 +325,28 @@ def split(doc_ids: list[int], pages: list[list[int]]):
         logger.exception(f"Error splitting document {doc.id}: {e}")
 
     return "OK"
+
+
+def delete_pages(doc_ids: list[int], pages: list[int]):
+    logger.info(
+        f"Attempting to delete pages {pages} from {len(doc_ids)} documents",
+    )
+    doc = Document.objects.get(id=doc_ids[0])
+    pages = sorted(pages)  # sort pages to avoid index issues
+    import pikepdf
+
+    try:
+        with pikepdf.open(doc.source_path, allow_overwriting_input=True) as pdf:
```

Comment:
Similar to split, I think remove_unreferenced_resources() should be called

Reply:
Ah thanks, yes!